src/allnotebuttontreedelegateeditor.cpp

```diff
@@ -76,8 +76,8 @@ void AllNoteButtonTreeDelegateEditor::paintEvent(QPaintEvent *event)
 #else
     int iconPointSizeOffset = -4;
 #endif
-    painter.setFont(FontLoader::getInstance().loadFont("Material Symbols Outlined", "",
-                                                       16 + iconPointSizeOffset));
+    painter.setFont(
+            font_loader::loadFont("Material Symbols Outlined", "", 16 + iconPointSizeOffset));
```

Comment:
Why use snake_case here?

Reply:
That's a setting in our .clang-tidy file: namespaces should be snake case. We can change this if you want, but personally I prefer that because it makes a clear distinction between namespaces and classes, without having to look at the implementation. If we would call it `FontLoader` instead, you'd think we're calling a static method in a class, and not a standalone function from a namespace.

Reply:
Gotcha 👍